server/index.js

```diff
@@ -18,63 +19,93 @@ app.use('*.js', (req, res, next) => {
 app.use(express.static('./client/dist'));
 app.use(express.json());
 
-/* ******** Personnel Routes ***************** */
+/* ******** Get Routes ***************** */
+
+app.get('/api/movies', async (req, res) => {
```

Comment:
I think you're better served batching logic for `/api/movies` in one place and `/api/personnel` in another place rather than by HTTP verbs.